outputs.tf

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "topic" {
+  value       = ["${google_pubsub_topic.topic.name}"]
```

Comment:
Does this value need to be a list, or can it just be the topic name?

Reply:
Exactly. I'm not sure why I have changed this into list. There is no reason to use list.
Fixed: https://github.com/terraform-google-modules/terraform-google-pubsub/pull/1/commits/c808032ba6ad5046407c771113555c9feec95007